tests/unit/docker/test_hub_build.py

```diff
@@ -59,6 +66,7 @@ def test_hub_build_push():
         '--keywords', summary['manifest_info']['keywords'][0],
         '--type', summary['manifest_info']['type']
     ])
+    monkeypatch.setattr(yaml, 'load', access_token_github)
```

Comment:
`JAML.load()` will get executed only if `credentials_file().is_file()` is True. Please patch that too